walkers/src/map.rs

```diff
@@ -186,7 +194,9 @@ impl Map<'_, '_, '_> {
 
             // Shift by 1 because of the values given by zoom_delta(). Multiple by 2, because
             // then it felt right with both mouse wheel, and an Android phone.
-            self.memory.zoom.zoom_by((zoom_delta - 1.) * 2.);
+            self.memory
+                .zoom
+                .zoom_by((zoom_delta - 1.) * 2. * self.zoom_speed);
```

Comment:
Would not be better if we move `2.` to be the default value (along with the comment)?

Reply:
Yeah that makes more sense, I'll update it with that